docs/UserGuide.md

```diff
@@ -204,9 +205,9 @@ Format: `list [c/CATEGORY [DESCENDING]]`
 * The `CATEGORY` is optional. By default, without stating the category, `list` will display all internships in no particular order
   * Possible options for `CATEGORY` : `company_name`, `position`, `application_process`, `date` (case-insensitive)
   * Case-insensitive: `company_name`, `Company_Name` are all acceptable inputs.
-* The `DESCENDING` tag is optional. It can take on the value `true` or `false` (case-insensitive). 
-  * The DESCENDING tag is set to `false` by default if not stated. List of internships will be shown in ascending order.
-  * The `DESCENDING` tag can only be set to `true` if the `CATEGORY` is stated
+* The `DESCENDING` parameter is optional. It can take on the value `true` or `false` (case-insensitive). 
+  * The DESCENDING parameter is set to `false` by default if not stated. List of internships will be shown in ascending order.
```

Comment:
I think it should be `DESCENDING` instead of DESCENDING

Reply:
Right, I will update this